orchagent/portsorch.cpp

```diff
@@ -2553,6 +2581,221 @@ bool PortsOrch::removeBridgePort(Port &port)
     return true;
 }
 
+bool PortsOrch::setHostVlanAdminState(int vlan_id, const string &admin_status)
+{
+    SWSS_LOG_ENTER();
+
+    /* The command should be generated as:
+ *      /sbin/ip link set Vlan{{vlan_id}} {{admin_status}}
+ *           */
+    const std::string cmds = std::string("")
+      + IP_CMD + " link set " + VLAN_PREFIX + std::to_string(vlan_id) + " " + admin_status;
+
+    std::string res;
+    EXEC_WITH_ERROR_THROW(cmds, res);
+
+    return true;
+}
+
+string PortsOrch::getPortOperStatus(string port_alias)
```

Comment:
Why  changing  "bool PortsOrch::getPortOperStatus(const Port& port, sai_port_oper_status_t& status) const"?

you return both boolean false and string,   could it pass build?

Reply:
This is a mistake，I'll fix it. Interestingly, it passed the build